src/coldfront_plugin_api/serializers.py

```diff
@@ -23,28 +28,72 @@ def get_status(self, obj: Project) -> str:
         return obj.status.name
 
 
+class AllocationAttributeSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = AllocationAttribute
+        fields = ["allocation_attribute_type", "value"]
+
+    allocation_attribute_type = serializers.SlugRelatedField(
+        read_only=False,
+        slug_field="name",
+        queryset=AllocationAttributeType.objects.all(),
+    )
+    value = serializers.CharField(read_only=False)
+
+
 class AllocationSerializer(serializers.ModelSerializer):
     class Meta:
         model = Allocation
-        fields = ["id", "project", "description", "resource", "status", "attributes"]
+        fields = [
+            "id",
+            "project",
+            "project_id",
+            "description",
+            "resource",
+            "resources_id",
+            "status",
+            "allocationattribute_set",
```

Comment:
Is there any way to "rename" how this fields appear in the output? I think `allocationattribute_set` is too "internal implementation".